### ISSUE TITLE
Disabling and enabling a plugin in Creation based games puts them in random positions.

### ISSUE THREAD

**Issue by Al12rs** [ModOrganizer2/modorganizer#650]
Needs investigation on older games as well.
Disabling and enabling a mod actually keeps the removed plugin at the same position and this behavior should be maintained after a fix.

**Comment by twizz0r** [comment]
Related? MO2 adds enabled plugins in alphabetical order rather than their priority in the left pane. 

![disabled](https://user-images.githubusercontent.com/36304651/52866468-b6567a00-310c-11e9-9dd6-e24842096fa1.png)

![enabled](https://user-images.githubusercontent.com/36304651/52866462-b2c2f300-310c-11e9-9b99-0338ef3fb889.png)

**Comment by LostDragonist** [comment]
No, that's not really related.  There is practically no relationship between the mod list and the plugin list.  There's no reason to expect new plugins to be added in any specific location or order.

**Comment by twizz0r** [comment]
K. If you check them individually you get order agreement, but when enabling multiples, you get alphabetical. No big. Just makes me rearrange plugins before merging. Would be small QOL improvement.
